### PR TITLE
Track deposits of dynamic validators

### LINKED ISSUE #267: Track deposits of dynamic validators

### PR DESCRIPTION
**Pull request by foriequal0**
Resolves #267

### CODE REVIEW

File: src/models/logic/cccChange.ts [hunk 1]
```diff
@@ -11,7 +11,13 @@ async function createCCCChange(
         change: U64;
         isNegative: boolean;
         blockNumber: number;
-        reason: "fee" | "author" | "stake" | "tx" | "initial_distribution";
+        reason:
```
Comment (sgkim126) [inline src/models/logic/cccChange.ts]: Why don't you use `Reason`?

**Comment by foriequal0** [comment]
This PR requires `chain_getCommonParams` or `chain_getTermMetadata` which is scheduled in 2.0.0

**Comment by sgkim126** [comment]
@foriequal0 I requested to create a docker image for the current master branch.
https://travis-ci.org/CodeChain-io/codechain/builds/551105731
Please update the docker tag after the image created.

**Comment by majecty** [comment]
Please check the worker/account.ts
The accounts whose balance is changed are updated at the file.
The accounts related the nomiation should be handled at the file.

**Comment by majecty** [comment]
If you modified the model type, you should create a migration file also.
Please change the CCCChanges model.

File: src/worker/cccChange.ts [hunk 2]
```diff
@@ -46,6 +52,7 @@ export async function updateCCCChange(
             transaction
         )
     );
+    queries.push(onTermClose(sdk, block.number));
```
Comment (majecty) [inline src/worker/cccChange.ts]: Please use the DB transaction.

**Comment by majecty** [comment]
@foriequal0 Could you fix the broken test cases?

**Comment by foriequal0** [comment]
I need to close and re-crate this PR on to the `codechain2` branch.

**Comment by foriequal0** [comment]
I didn't know I can change the base branch. Yay!
image

**Pull request merged**